src/lib/storage/proposal/space_maker.rb

```diff
@@ -205,15 +205,17 @@ def resize_windows!(volumes, disk, force: false)
           parts_by_disk = partitions_by_disk(part_names)
           remove_linux_disks!(parts_by_disk) unless force
 
-          sorted_resizables(parts_by_disk.values.flatten).each do |res|
+          success = sorted_resizables(parts_by_disk.values.flatten).any? do |res|
             shrink_size = [
               res[:recoverable_size],
               missing_required_size(volumes, disk)
             ].min
             shrink_partition(res[:partition], shrink_size)
-            return if success?(volumes)
+
+            success?(volumes)
           end
-          log.info "Didn't manage to free enough space by resizing Windows"
+
+          log.info "Didn't manage to free enough space by resizing Windows" unless success
```

Comment:
btw also `sorted_resizables(parts_by_disk.values.flatten)` is not nice api, I expect something like `disks.partitions_with_windows.reduce(:+).sort_by(&:size).any?` which is more readable for me what it actually probably do ( give me disks, their windows partitions and sort it by size )